src/iotjs_context.c

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2015-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "iotjs_context.h"
+
+#ifndef IOTJS_ENABLE_EXTERNAL_CONTEXT
+
+/**
+ * Global context.
+ */
+iotjs_context_t iotjs_global_context;
```

Comment:
This global variable should be initialized. Otherwise, OSX builds will link-fail. (Symbol `iotjs_global_context` will become a common symbol instead of a data section symbol.)